account_invoice_payment_term_date_due/views/account_move.xml

```diff
@@ -5,23 +5,23 @@
     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-
     <record id="view_move_form" model="ir.ui.view">
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
-            <field name="invoice_payment_term_id" position="after">
-                <span
-                    name="invoice_payment_term_date_due"
-                    class="text-muted oe_read_only"
+            <xpath expr="//field[@name='invoice_payment_term_id']" position="after">
+                <i
+                    class="fa fa-arrow-right mx-3 text-muted"
                     attrs="{'invisible': ['|', ('invoice_payment_term_id', '=', False), ('invoice_date_due', '=', False)]}"
-                    t-translation="off"
-                >
-                    <i class="fa fa-arrow-right mx-3" />
-                    <field name="invoice_date_due" />
-                </span>
-            </field>
+                    style="padding-left: 30px;padding-top: 2px;"
```

Comment:
Generally is better to avoid inline styles like this one.
Can't it be solved with some boostrap classes instead? (ie: `pl-2 pt-1` or something like that)

Reply:
Perfect! I added the `pl-4 pt-1` like you suggested. It is displayed correctly.